modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/NumericDDMFormFieldTypeUtil.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.field.type.internal.util;
+
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.dynamic.data.mapping.render.DDMFormFieldRenderingContext;
+import com.liferay.dynamic.data.mapping.util.NumericDDMFormFieldUtil;
+import com.liferay.portal.kernel.json.JSONException;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class NumericDDMFormFieldTypeUtil {
```

Comment:
Yes. I was avoiding duplicate code in `ValidationDDMFormFieldTemplateContextContributor` ([here](https://github.com/liferay-forms/liferay-portal/pull/1291/commits/8b564308ee23516ba3630b70e2ad92c22c74d107#diff-ec279721cb20e911110bd3b32fdbf716d2f911888f5f84e744bccdbf316aa3cbR62-R64) to be more specific).

Reply:
That makes sense, @carolmariaabb. Thanks for clarifying.
I'll just review the moved code now, because, even though you're just moving stuff, Brian reviews it as a change and an opportunity to comment on old decisions.